README.md

```diff
@@ -0,0 +1,29 @@
+# Operator Manifest Verification
+Operator Manifest Verification library (`operator-verify`) verifies static operator files against [Operator-Lifecycle-Manager's](https://github.com/operator-framework/operator-lifecycle-manager) types (`ClusterServiceVersions`, `CustomResourceDefinitions`, and `Package` Manifest yamls).
+
+Currently, it supports validation of `ClusterServiceVersion` yaml for any mismatched data types with Operator-Lifecycle-Manager's `ClusterServiceVersion` [type](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/pkg/api/apis/operators/v1alpha1/clusterserviceversion_types.go#L359:6). 
+
+# Getting Started
+
+The Operator Manifest Verfication library defines a single definition of a valid operator. It helps in validating operator manifest bundles before deploying them on cluster, and thus, helping in the operator development process. 
+
+The operator manifest bundles are an amalgamation of [ClusterServiceVersions](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/design/building-your-csv.md), [CustomResourceDefinitions](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/), and [Package Manifest](https://github.com/operator-framework/operator-lifecycle-manager#discovery-catalogs-and-automated-upgrades) yamls.
+
+# Usage
+
+Currently, you can use this library with the following Command Line Tool:
+
+## Command Line Tool
+You can install the `operator-verify` tool from source using:
+
+`go install`
+
+Check `echo $PATH`. If you do not have `$GOPATH/bin` in your `$PATH`, 
```

Comment:
Explain why we need this. (i.e. because otherwise you won't be able to call `operator-verify` anywhere on your machine)